bears/general/URLBear.py

```diff
@@ -0,0 +1,155 @@
+import re
+import requests
+from urllib.parse import urlparse
+from aenum import Flag
+
+from coalib.bears.LocalBear import LocalBear
+from dependency_management.requirements.PipRequirement import PipRequirement
+from coalib.bearlib import deprecate_settings
+from coalib.results.HiddenResult import HiddenResult
+from coalib.settings.Setting import typed_list, typed_dict
+from coalib.parsing.Globbing import fnmatch
+
+
+class LINK_CONTEXT(Flag):
+    no_flags = 0
+    xml_namespace = 1
+    pip_vcs_url = 2
+    all_flags = 3
+
+
+class URLBear(LocalBear):
+    DEFAULT_TIMEOUT = 15
+    LANGUAGES = {'All'}
+    REQUIREMENTS = {PipRequirement('requests', '2.12'),
+                    PipRequirement('aenum', '2.0.8')}
+    AUTHORS = {'The coala developers'}
+    AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
+    LICENSE = 'AGPL-3.0'
+    CAN_DETECT = {'Documentation'}
+
+    # IP Address of www.google.com
+    check_connection_url = 'http://216.58.218.174'
```

Comment:
when this is merged, create an issue to make `URLBear.check_connection_url` a setting instead of a hidden constant